tests/test_table_transformations.py

```diff
@@ -38,10 +38,10 @@ def test_result_data_shape_basic_with_sorting(self):
 
         self.assertIsNotNone(df)
         pdt.assert_frame_equal(df, pd.DataFrame([
-            ['P1', 'D1', 'BS1', 'BM1', 42, 'Diagnosis 1 Name', 'Skin', '2018-04-24'],
-            ['P1', 'D1', 'BS1', 'BM2', 42, 'Diagnosis 1 Name', 'Skin', '2018-03-07'],
-            ['P2', 'D1', 'BS2', 'BM3', 39, 'Diagnosis 1 Name 2', 'Liver', '2018-01-19'],
-            ['P2', 'D1', 'BS2', 'BM4', 39, 'Diagnosis 1 Name 2', 'Liver', '2011-06-05'],
+            ['P1', 'D1', 'BS1', 'BM1', 42., 'Diagnosis 1 Name', 'Skin', '2018-04-24T02:00:00Z'],
+            ['P1', 'D1', 'BS1', 'BM2', 42., 'Diagnosis 1 Name', 'Skin', 'Wed Mar 07 01:00:00 CET 2018'],
+            ['P2', 'D1', 'BS2', 'BM3', 39., 'Diagnosis 1 Name 2', 'Liver', 'Fri Jan 19 01:00:00 CET 2018'],
+            ['P2', 'D1', 'BS2', 'BM4', 39., 'Diagnosis 1 Name 2', 'Liver', 'Sun Jun 05 02:00:00 CEST 2011'],
```

Comment:
why the dates are not formatted any more?

Reply:
Formatting happens in `format_columns` now. Wich is called by `from_obs_json_to_formatted_pdbb_df`.

Reply:
Just to separate concerns and test them separatly.

Reply:
ahh ok! did not notice the separation